test/e2e/utils/image/manifests.go

```diff
@@ -0,0 +1,49 @@
+// Copyright (c) 2023 ScyllaDB.
+
+package image
+
+import "fmt"
+
+// RegistryList holds public and private image registries
+type RegistryList struct {
+	DockerLibraryRegistry string `json:"dockerLibraryRegistry"`
+}
+
+// Config holds an images registry, name, and version
+type Config struct {
+	registry string
+	name     string
+	version  string
+}
+
+var (
+	registry = initRegistry()
+
+	imageConfigs = initImageConfigs(registry)
```

Comment:
What's the point of this package? It seems like a bit of a convoluted logic to get a hardcoded image reference string in the end?

Reply:
fwiw, kube has it so the downstreams can customize universal images (like for bash, so they play well with their security, like user ids). I don't think we have the need but I don't feel strong one way or the other